packages/image-block/src/components/Image.tsx

```diff
@@ -39,26 +39,34 @@ export const Image = ({ image, appBridge, blockSettings, isEditing }: ImageProps
                     target={link.openInNewTab ? '_blank' : undefined}
                     rel="noreferrer"
```

Comment:
```suggestion
                    rel={link.openInNewTab ? 'noopener noreferrer' : 'noreferrer'}
```

Add also noopener if its target is _blank to prevent tabnapping

Reply:
great suggestion, added